<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>242</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>242</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/resources/Luma.ico</normaloff>:/images/resources/Luma.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>242</width>
      <height>340</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabGen">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="Switch" name="sSC" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbMs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Mouse</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbKb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Keyboard</string>
      </property>
      <widget class="QLabel" name="lKb">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>QWERTY keyboard</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Shortcuts</string>
       </property>
      </widget>
     </widget>
     <widget class="Switch" name="sInvX" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lInvX">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Invert X axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="lInvY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Invert Y axis</string>
      </property>
     </widget>
     <widget class="Switch" name="sInf" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lDis">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
     </widget>
     <widget class="Switch" name="sDis" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="Switch" name="sKb" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="Switch" name="sInvY" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lInf">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Infinite screen</string>
      </property>
     </widget>
     <zorder>gbKb</zorder>
     <zorder>sSC</zorder>
     <zorder>gbMs</zorder>
     <zorder>sInvX</zorder>
     <zorder>lInvX</zorder>
     <zorder>lInvY</zorder>
     <zorder>sInf</zorder>
     <zorder>lDis</zorder>
     <zorder>sDis</zorder>
     <zorder>sKb</zorder>
     <zorder>sInvY</zorder>
     <zorder>lInf</zorder>
    </widget>
    <widget class="QWidget" name="tabShortcut">
     <attribute name="title">
      <string>Shortcut</string>
     </attribute>
     <widget class="QLabel" name="lSC">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Active shortcuts</string>
      </property>
     </widget>
     <widget class="QListView" name="liSC">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>40</y>
        <width>222</width>
        <height>221</height>
       </rect>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="bAdd">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>270</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bDel">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>270</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QTreeView" name="view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSound">
     <attribute name="title">
      <string>sound</string>
     </attribute>
     <widget class="QComboBox" name="cMic">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>80</y>
        <width>180</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="Switch" name="sLoopBack" native="true">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lAct">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Activate mic loopback</string>
      </property>
     </widget>
     <widget class="QLabel" name="lMic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mic:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume:</string>
      </property>
     </widget>
     <widget class="QSlider" name="slVol">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>160</y>
        <width>180</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Switch</class>
   <extends>QWidget</extends>
   <header location="global">switch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
